### DISCUSSION TITLE
Web API failed on POST

### DISCUSSION THREAD

**Discussion by DenisMtfl** [tbnobody/OpenDTU#2102] [Q&A]
With Postman or cURL I got the reponse "No values found" on POSTs.
Whats wrong?

C:\Users\denis>curl -u "admin:xxxx" http://192.168.1.160/api/ntp/config -d 'data={"ntp_server":"my.own.ntp.server
.home","ntp_timezone":"CET-1CEST,M3.5.0,M10.5.0/3","ntp_timezone_descr":"Europe/Berlin"}'
{"type":"warning","message":"No values found!","code":1002}"

**Comment by ms49434** [comment]
Please add the keys for longitude, latitude and sunsettype to POST. See WebApi_ntp.cpp, method  onNtpAdminPost()
